samtranslator/model/sam_resources.py

```diff
@@ -2392,3 +2397,9 @@ def _construct_ddb_datasource_connector_resources(
             SamConnector(logical_id=logical_id).from_dict(logical_id=logical_id, resource_dict=connector_dict),
         )
         return connector.to_cloudformation(**kwargs)
+
+    @staticmethod
+    def _set_resolver_code_settings(resolver_code_settings: aws_serverless_graphqlapi.ResolverCodeSettings) -> None:
+        # TODO: Add FileNamePatterns defaults once implemented
+        resolver_code_settings.FunctionsFolder = resolver_code_settings.FunctionsFolder or "functions"
+        resolver_code_settings.ResolversFolder = resolver_code_settings.ResolversFolder or "resolvers"
```

Comment:
What's a "folder" technically here? S3 bucket? Local path? "Folder" is also quite a Windows-y term, so not sure if that's ideal.

Reply:
It would be the prefix where customers store their functions. Example:

`PathToCode: s3://graph-ql-demo/`
`FunctionsFolder: functions`

So the user's functions would be stored with the s3 path prefix `s3://graph-ql-demo/functions/`.

Even in S3, these subsections are referred to as [folders](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html) (although they are not exactly the same since S3 is a flat structure vs. a tree like file structure). So I think `Folder` is fine.

Reply:
Why not just make it a bucket name? Less URI verbosity and easy to get with intrinsic.

Reply:
Actually closer to a URI than just “folder” due to its unmistakable URI format.